src/utils/filterVideo.js

```diff
@@ -0,0 +1,23 @@
+function filteredVideosOnCategory(selectedCategory, videos) {
+  let filterVideos = videos;
```

Comment:
Just nitpicking. Instead of naming the variable as filterVideos, I guess filteredVideos would be more suitable.

Reply:
Yupp, I will change that